webapp/channels/src/components/announcement_bar/default_announcement_bar/announcement_bar.tsx

```diff
@@ -196,10 +194,8 @@ export default class AnnouncementBar extends React.PureComponent<Props, State> {
         if (this.state.showTooltip) {
             barContent = (
                 <WithTooltip
-                    id='announcement-bar__tooltip'
                     title={this.props.tooltipMsg ? this.props.tooltipMsg : message}
-                    placement='bottom'
-                    delayHide={this.state.isStringContainingUrl ? OVERLAY_ANNOUNCEMENT_HIDE_DELAY : 0}
```

Comment:
Have you tested this? This delay here was to make tooltips with a link easier to click on the link. If they disappear too fast, it may not let you get the mouse to the correct place to click the link.

Reply:
oh thank you for the context behind this one. Unfortunately if we are going to have link in the tooltip then we should not be using a tooltip for this, see https://css-tricks.com/tooltip-best-practices/

But for the scope of the PR what do you propose, add yet another config prop to manage the delay or we solve it separately (Which isnt a big effort i would say)?

Reply:
One part of me wants to remove this delay altogether, and I "tried" when I first implemented `WithTooltip`. But for some reason, at that moment, made more sense to keep it (probably just not to break anything).

I am fine keeping the prop (maybe a boolean instead? 0/5), specially not to "dirty" this PR. But if you have a good solution for this that includes removing the prop, you would make me very happy :smile:

Reply:
I do but unfortunately, I cannot add that yet, you will have to continue to add that provision temporarily here.